### ISSUE TITLE
Training on FlyingChairs

### ISSUE THREAD

**Issue by julien-mille** [sniklaus/pytorch-liteflownet#27]
Hi Simon,
In parallel to translating Hui's pretrained Caffe model to Pytorch, did you try to train your LiteFlowNet implementation from scratch?
I tried to train it (from scratch, end-to-end) on FlyingChairs using the Adam optimizer with different learning rates (lr=0.01, lr=0.001) but could not have the L2 loss decrease properly (neither on the trainset nor the val set).
Thanks for any advice.
Julien

**Comment by sniklaus** [comment]
Thank you for your interest in this repository! I am afraid that it focuses on replicating the inference of the official LiteFlowNet implementation though, and it does not make any efforts to replicate the training. My apologies that I will thus not be able to assist you in your training efforts.

**Comment by julien-mille** [comment]
No problem. Thanks for your quick response
